patches/server/0184-Fix-Citizens.patch

```diff
@@ -65,42 +65,43 @@ index df163434fc5245d893d23ac375bc5f27d1315d4f..4d2f9662d4e803fccaca39f7356f5e0c
  
      public static boolean isExternalPlayer(Entity entity) {
 -        return entity instanceof ExternalPlayer;
-+        return entity instanceof ExternalPlayer && !entity.getBukkitEntity().hasMetadata("NPC");
++        return entity instanceof ExternalPlayer && !entity.getBukkitEntity().hasMetadata("NPC"); // MultiPaper - DoctaEnkoda Fix Citizens Check
      }
  
      public static boolean isExternalPlayer(org.bukkit.entity.Entity bukkitEntity) {
 diff --git a/src/main/java/puregero/multipaper/MultiPaperEntitiesHandler.java b/src/main/java/puregero/multipaper/MultiPaperEntitiesHandler.java
-index ff7c1726c323a104bac9a9a8b3a8ea6d6414dc75..00b4421b94c34b9cc71011806bb146a1de3cca2d 100644
+index ff7c1726c323a104bac9a9a8b3a8ea6d6414dc75..7ef7682efaca41154f3f9fb3dacd373098900fb2 100644
 --- a/src/main/java/puregero/multipaper/MultiPaperEntitiesHandler.java
 +++ b/src/main/java/puregero/multipaper/MultiPaperEntitiesHandler.java
-@@ -349,7 +349,14 @@ public class MultiPaperEntitiesHandler {
+@@ -349,7 +349,15 @@ public class MultiPaperEntitiesHandler {
          if (entity instanceof ExternalPlayer) return;
          if (reason == Entity.RemovalReason.UNLOADED_TO_CHUNK || reason == Entity.RemovalReason.UNLOADED_WITH_PLAYER) return;
  
 -        LevelChunk chunk = entity.level.getChunkIfLoaded(entity.blockPosition());
++        // MultiPaper - DoctaEnkoda Start Fix Citizens
 +        LevelChunk chunk;
 +
 +        if (reason == Entity.RemovalReason.KILLED) {
 +            chunk = entity.level.getChunkAt(entity.blockPosition);
 +        } else {
 +            chunk = entity.level.getChunkIfLoaded(entity.blockPosition());
 +        }
-+
++        // MultiPaper - DoctaEnkoda End Fix Citizens
          if (chunk != null) {
              MultiPaper.broadcastPacketToExternalServers(chunk.externalEntitiesSubscribers, () -> new EntityRemovePacket(entity));
          } else {
 diff --git a/src/main/java/puregero/multipaper/externalserverprotocol/RequestEntityPacket.java b/src/main/java/puregero/multipaper/externalserverprotocol/RequestEntityPacket.java
-index 4f27d37e3fbdf1570f907d1eb5eaa24ea6d638c5..b6a71b69c6f7c32b21b02611a0923018645011ba 100644
+index 4f27d37e3fbdf1570f907d1eb5eaa24ea6d638c5..5087ef4ca4e4662d1ee05b976382b8d2e8ce4bab 100644
 --- a/src/main/java/puregero/multipaper/externalserverprotocol/RequestEntityPacket.java
 +++ b/src/main/java/puregero/multipaper/externalserverprotocol/RequestEntityPacket.java
 @@ -70,6 +70,10 @@ public class RequestEntityPacket extends ExternalServerPacket {
      }
  
      private void triedToRequestPlayer(ExternalServerConnection connection, ServerPlayer serverPlayer) {
-+        // DoctaEnkoda - Start Fix Citizens
++        // MultiPaper - DoctaEnkoda Start Fix Citizens
```

Comment:
`puregero.multipaper` package does not need `// MultiPaper ` comments as its project the classes, 

for example we changed paper chunk system we add comments but if its project stuff like package mentioned above no need for it.

Reply:
I will remember to delete it in the future when I submit, thanks for your help :)